pkg/operator/controllers/node/node_controller_test.go

```diff
@@ -0,0 +1,187 @@
+package node
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"reflect"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestIsDraining(t *testing.T) {
+	for _, tt := range []struct {
+		name string
+		node *corev1.Node
+		want bool
+	}{
+		{
+			name: "isDraining false",
```

Comment:
nit: name of test is misleading. Yes it is false but it is false because the node is unschedulable.  This seems to represent the same test on line 59

Reply:
This test covers us on this line: https://github.com/Azure/ARO-RP/blob/master/pkg/operator/controllers/node/node_controller.go#L187 and I am not sure how to rename it. Something like `isDraining false, no conditions are met`?

Reply:
If you were going to rename it to the above, I would create the test to check all the conditions that are not being met to satisfy the false on line 187.  This way you'll satisfy the test named `isDraining false, no conditions met`